public/data/FahimFBA.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Md. Fahim Bin Amin",
+    "bio": "Microsoft Research Investigation Contributor to OSS || Competitive Programmer || Tech-enthusiasts || Student of Computer Science and Engineering at United International University",
```

Comment:
```suggestion
    "bio": "Microsoft Research Investigation || Contributor to OSS || Competitive Programmer || Tech-enthusiasts || Student of Computer Science and Engineering at United International University",
```

Reply:
Can you please check that out now? I did the change afterward.